index.d.ts

```diff
@@ -63,14 +64,10 @@ import elementReady = require('element-ready');
 })();
 ```
 */
-declare function elementReady<ElementName extends keyof HTMLElementTagNameMap>(
-	selector: ElementName,
+declare function elementReady<Selector extends string, ElementName extends Element = ParseSelector<Selector>>(
```

Comment:
Should the return type default to `HTMLElement` instead of plain `Element`, [like `select-dom` does](https://github.com/fregante/select-dom/blob/93c48807aa37333a1b89c962f19863f151a3aade/index.ts#L18)?

Reply:
Yes. This also should apply in refined GitHub if it has been implemented otherwise (fetchDom?)